server/src/modules/mentorship/mentorship.model.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  Model,
+  Column,
+  Table,
+  DataType,
+  ForeignKey,
+  PrimaryKey,
+  AutoIncrement,
+} from 'sequelize-typescript';
+import { User } from '../user/user.model';
+
+@Table({
+  tableName: 'mentorship',
+})
+export class Mentorship extends Model<Mentorship> {
+  private static ACTIVE = 'Active';
+  private static COMPLETED = 'Completed';
+
+  private static MENTORSHIP_STATUS = DataType.ENUM(
+    Mentorship.ACTIVE,
+    Mentorship.COMPLETED,
```

Comment:
What's the difference between an ACTIVE/COMPLETED status and comparing the current date to the start/end date? I would assume the mentorship is active exactly between the start and end, so I don't understand what this is for.

Reply:
Yeah this is redundant.